src/trackers/classes/request.js

```diff
@@ -16,6 +18,18 @@ class Request {
         this.originalSubdomain = undefined
         this.responseHash = reqData.responseBodyHash
         this.nameservers = []
+        this.firstPartyCookies = site.documentCookies
+            .filter(cookie => cookie.source === reqData.url && (!cookie.domain || cookie.domain.endsWith(site.domain)) && cookie.value && cookie.value.length > COOKIE_LENGTH_CUTOFF)
+        this.firstPartyCookiesSent = site.documentCookies
+            .filter(cookie => {
+                // only consider cookies 5 or more characters long
+                if (cookie.value && cookie.value.length > COOKIE_LENGTH_CUTOFF) {
+                    // for longer cookie values, exclude the first 6 characters (GA cookies only send the suffix)
+                    const cookieSuffix = cookie.value.length > 10 ? cookie.value.slice(6) : cookie.value
+                    return this.url.indexOf(cookieSuffix) !== -1
```

Comment:
ohhhh, ok we are looking for cookie value in the url of all request, got it. Might be worth dropping a comment TBH. Couple of thoughts here:

1. looking for a cookie value in the whole url seems like something that can produce false possitives maybe it'd be better to limit the search to path & query of an url?
2. can we limit this GA-specific mitigation to GA only? e.g. only exclude those 6 chars if cookie name is `_ga`? For other cookies this may create false positives.

Reply:
1. True, I purposely ignored that issue here, because to do this properly it would be nice to have the URL parsing changes I proposed (https://app.asana.com/0/1128329641547046/1199565593892363/f).
 2. Sure, that makes sense.

Reply:
I've added a commit to:
 1. Only look on the url path and search params for the value
 2. Make specific handling for cookies (e.g. `_ga`) pluggable
 3. Test these things.